core/src/test/java/com/linecorp/armeria/client/endpoint/dns/DnsNameEncoder.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2018 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.client.endpoint.dns;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.handler.codec.dns.DefaultDnsRecordEncoder;
+
+final class DnsNameEncoder {
+
+    static void encodeName(String name, ByteBuf out) {
+        DefaultDnsRecordEncoderTrampoline.INSTANCE.encodeName(name, out);
+    }
+
+    private DnsNameEncoder() {}
+
+    // Hacky trampoline class to be able to access encodeName
```

Comment:
I forget if I suggested making this accesible from netty code, but now that we've been making changes there wonder if we can modify the API to avoid this hack